### DISCUSSION TITLE
How to get the xyz size of polydata?

### DISCUSSION THREAD

**Discussion by wociety** [pyvista/pyvista#4100] [Q&A]
Hello, I am a newbie.
Is there a simple way to get xyz size of a polydata? (like '.dimensions' of bpy)
What I am using now is like this.

```python
import pyvista as pv
model = pv.PolyData('mymodel.stl').clean()
xmin, xmax, ymin, ymax, zmin, zmax = model.bounds
xsize = xmax - xmin
ysize = ymax - ymin
zsize = zmax - zmin
```

**Comment by tkoyama010** [comment]
Thanks for using PyVista!
PyVista does not have a request method. The closest is [length](https://docs.pyvista.org/api/core/_autosummary/pyvista.DataSet.length.html), which finds the length of the diagonal. But I guess this is not what you want. We may create a new method, but the script you sent us is simple enough in my opinion.
